test/rockpaperscissors.js

```diff
@@ -0,0 +1,717 @@
+
+        assert.strictEqual(playerOneOwed.toString(10), "0");
+        assert.strictEqual(playerTwoOwed.toString(10), "20");
+
+    });
+
+    it("Player two does not reveal and player one collects forfeit", async () => {
+
+        await rcp.movePlayerOne(gameKey, toBN(10), {from: playerOne, value: toBN(10)});
+        await rcp.movePlayerTwo(gameKey, playerTwoMoveHash, {from: playerTwo, value: toBN(10)});
+        await rcp.revealPlayerOne(playerOneSecretBytes32, ROCK, {from: playerOne});
+        await timeMachine.advanceTimeAndBlock(DAY_24_HOUR_IN_SECS);
+
+        const txObj = await rcp.playerOneCollectsForfeit(gameKey, {from: playerOne});
+
+        await truffleAssert.eventEmitted(txObj, 'ForfeitPaid', (ev) => {
+
+            return  ev.player === playerOne &&
+                    ev.gameKey === gameKey &&
+                    ev.amount.toString(10) === "20"
+        },'ForfeitPaid event is emitted');
+
+        const playerOneOwed = await rcp.balances(playerOne);
+        const playerTwoOwed = await rcp.balances(playerTwo);
+
+        assert.strictEqual(playerOneOwed.toString(10), "20");
+        assert.strictEqual(playerTwoOwed.toString(10), "0");
+
+    });
+
+    it("Player two can stake their winnings in a subsequent game", async () => {
+        await rcp.movePlayerOne(gameKey, toBN(10), {from: playerOne, value: toBN(10)});
+        await rcp.movePlayerTwo(gameKey, playerTwoMoveHash, {from: playerTwo, value: toBN(10)});
+        await rcp.revealPlayerOne(playerOneSecretBytes32, ROCK, {from: playerOne});
+        await rcp.revealPlayerTwo(gameKey, playerTwoSecretBytes32, PAPER, {from: playerTwo});
+
+        const initPlayerTwoOwed = await rcp.balances(playerTwo);
+        assert.strictEqual(initPlayerTwoOwed.toString(10), "19");
+
+        const playerMoveHash = await rcp.createPlayerOneMoveHash(playerOneSecretBytes32, ROCK, {from: playerTwo});
+        await rcp.movePlayerOne(playerMoveHash, toBN(10), {from: playerTwo});
+
+        const playerTwoOwed = await rcp.balances(playerTwo);
+        assert.strictEqual(playerTwoOwed.toString(10), "9");
+
+    });
+
+    it("Player ends the game early, collects the stake and withdraws their balance", async () => {
+
+        await rcp.movePlayerOne(gameKey, toBN(10), {from: playerOne, value: toBN(10)});
+
+        const txObj = await rcp.playerOneEndsGame(gameKey, {from: playerOne});
+
+        const initPlayerOneEthBalance = toBN(await getBalance(playerOne));
+
+        await truffleAssert.eventEmitted(txObj, 'PlayerOneEndsGame', (ev) => {
+
+            return  ev.player === playerOne &&
+                    ev.gameKey === gameKey &&
+                    ev.amount.toString(10) === "10"
+        },'PlayerOneEndsGame event is emitted');
+
+        const playerOneOwed = await rcp.balances(playerOne);
+        assert.strictEqual(playerOneOwed.toString(10), "10");
+
+        // Recipient withdraws
+        const txObj2 = await rcp.withdraw(toBN(5), {from: playerOne});
```

Comment:
You ought to test only 1 thing per `it`. Here you tested the result of `rcp.playerOneEndsGame` (which should be in another `it`) and now move on to testing `rcp.withdraw`, which is ok.